### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by jesper-friis**

### CODE REVIEW

File: doc/Doxyfile.in [hunk 1]
```diff
@@ -44,14 +44,14 @@ PROJECT_NUMBER         = "@dlite_VERSION@"
 # for a project that appears at the top of each page and should give viewer a
 # quick idea about the purpose of the project. Keep the description short.
 
-PROJECT_BRIEF          = "Lightweight library for working with scientific data"
+PROJECT_BRIEF          = "Lightweight data-centric interoperability framework for working with scientific data"
```
Comment (CasperWA) [inline doc/Doxyfile.in:47]: This should be updated in `python/setup.py` as well for the short description - and anywhere else that's similar.

**Review by CasperWA — COMMENTED** [review]
It looks good to me, but then I don't know exactly how to build this myself to check it.

Could you add a small section in the README with instructions on how to build the documentation locally?

File: doc/Doxyfile.in (hunk 1 above)
Comment (quaat) [reply doc/Doxyfile.in:47]: I am not sure DLite is limited to scientific data. How about branding it as:
"DLite - A lightweight data-centric framework for semantic interoperability"?

Comment (quaat) [reply doc/Doxyfile.in:47]: I am not sure DLite is limited to scientific data. How about branding it as:
"DLite - A lightweight data-centric framework for semantic interoperability"?

**Review by quaat — APPROVED** [review]

File: doc/Doxyfile.in (hunk 1 above)
Comment (jesper-friis) [reply doc/Doxyfile.in:47]: Done

**Comment by jesper-friis** [comment]
> It looks good to me, but then I don't know exactly how to build this myself to check it.
> 
> Could you add a small section in the README with instructions on how to build the documentation locally?

Done

**Pull request merged**